baselines/ilswiss_minimal/setup.py

```diff
@@ -0,0 +1,17 @@
+from setuptools import find_packages, setup
+
+setup(
+    name="ilswiss_minimal",
+    description="Minimal Version of ILSwiss Designed for SMARTS",
+    version="0.1.0",
+    packages=find_packages(exclude=["tests"]),
+    include_package_data=True,
+    zip_safe=True,
+    python_requires=">=3.6",
+    install_requires=[
+        "smarts[train]",
+        "black==20.8b1",
+        "gtimer",
+        "gym",
```

Comment:
To fix the dependency versions, kindly specify the versions used for `smarts[train]`, `gtimer`, and `gym` dependencies.